### ISSUE TITLE
set.intersection help suggests 2 sets, while any number > 0 is supported

### ISSUE THREAD

**Issue by bwagner** [python/cpython#99967]
# set.intersection help suggests 2 sets, while any number > 0 is supported

`pydoc set.intersection` states:
 
> set.intersection = intersection(...)
>     Return the intersection of **two** sets as a new set.
> 
>  (i.e. all elements that are in **both** sets.)
> 

while in fact one or more sets can be passed to the function, e.g.
```pycon
>>> set.intersection({0,1,2,3}, {1,2,3,4}, {2,3,4,5})
{2, 3}
>>> set.intersection({0,1,2,3})
{0, 1, 2, 3}
```

**Comment by ramvikrams** [comment]
i'll do it

**Comment by rhettinger** [comment]
Here are some proposed edits.  I'll add more shortly:

```
union_doc:
Return the union of this set with zero or more iterables.

An element is in the union set if and only if it occurs
in this set or in one of the input iterables.


intersection_doc:
Return the intersection of this set with zero or more iterables.

An element is in the intersection set if and only if it occurs
in this set and in all of the input iterables.


difference_doc:
Return the difference (complement) of this set with zero or more iterables.

An element is in the intersection set if and only if it occurs
in this set but not in any of the input iterables.
```

**Comment by ramvikrams** [comment]
> Here are some proposed edits. I'll add more shortly:
> 
> ```
> union_doc:
> Return the union of this set with zero or more iterables.
> 
> An element is in the union set if and only if it occurs
> in this set or in one of the input iterables.
> 
> 
> intersection_doc:
> Return the intersection of this set with zero or more iterables.
> 
> An element is in the intersection set if and only if it occurs
> in this set and in all of the input iterables.
> 
> 
> difference_doc:
> Return the difference (complement) of this set with zero or more iterables.
> 
> An element is in the intersection set if and only if it occurs
> in this set but not in any of the input iterables.
> ```

but are you not doing the changes

**Comment by rhettinger** [comment]
Yes. I'll do the changes want to work out the edits here first.  It is a process.

**Comment by ramvikrams** [comment]
> Yes, I'll do the changes but want to work out the edits here first. It is a process.

oh, thanks i didn't know about it.
